kelim_shluvim.py

```diff
@@ -0,0 +1,12 @@
+import itertools
+
+
+def interleave(*iterables: iter):
+    """
+    THe function get iterables and merge them alternately
+    :param iterables: iterables objects to merge
```

Comment:
Here you use iterables as an adjective so it shouldnt be written in plurals (iterable objects to merge.)